integration-tests/hibernate-reactive-mssql/README.md

```diff
@@ -0,0 +1,47 @@
+# JPA example with Microsoft SQL Server
```

Comment:
IMO we should either remove the README or update it to talk about HR instead of JPA.

Reply:
I would drop it.